alert/meta.go

```diff
@@ -0,0 +1,26 @@
+package alert
+
+const TypeAlertMetaV1 = "alert_meta_v1"
+
+type AlertMeta struct {
+	Type        string        `json:"type"`
+	AlertMetaV1 AlertMetaData `json:"alert_meta_v1"`
+}
```

Comment:
These types should be private, and are only used going to/from the DB layer

Reply:
removed the json typing